fairroot/base/sim/FairModule.h

```diff
@@ -137,14 +137,14 @@ class FairModule : public TNamed
     TList* GetListOfGeoPar() { return flGeoPar; }
 
     /**list of volumes in a simulation session*/
-    static thread_local FairVolumeList* vList;   //!
+    static thread_local inline FairVolumeList* vList{0};   //!
```

Comment:
Can you please use `nullptr` instead of `0`?

```suggestion
    static thread_local inline FairVolumeList* vList{nullptr};   //!
```